src/components/Header/index.jsx

```diff
@@ -1,18 +1,32 @@
-import React from "react";
+import React, { useState } from "react";
 import styles from "./styles.module.css";
 import classNames from "classnames";
 import PropTypes from "prop-types";
 import Logo from "components/Logo";
 import logoSrc from "assets/circle.svg";
 import { Link } from "react-router-dom";
+import HamburgerButton from "components/HamburgerButton";
+import Sidebar from "components/Sidebar";
+
+const Header = ({ className, _routes }) => {
+  const [toggle, setToggle] = useState(false);
 
-const Header = ({ className }) => {
   return (
-    <div className={classNames(className, styles.header)}>
-      <Link to="/">
-        <Logo imageSrc={logoSrc} />
-      </Link>
-    </div>
+    <>
```

Comment:
I don't think you need fragments if you already have one div wrapping everything.

Reply:
Yeah but if you look closely there is another adjacent div and last I checked adjacent divs require wrapping. You can find more information on the concept at reactjs.org 🧐 .